cluster-operator/src/main/java/io/strimzi/operator/cluster/model/CruiseControl.java

```diff
@@ -91,15 +92,34 @@ public class CruiseControl extends AbstractModel implements SupportsMetrics, Sup
     protected static final String TLS_CC_CERTS_VOLUME_MOUNT = "/etc/cruise-control/cc-certs/";
     protected static final String TLS_CA_CERTS_VOLUME_NAME = "cluster-ca-certs";
     protected static final String TLS_CA_CERTS_VOLUME_MOUNT = "/etc/cruise-control/cluster-ca-certs/";
-    protected static final String LOG_AND_METRICS_CONFIG_VOLUME_NAME = "cruise-control-metrics-and-logging";
-    protected static final String LOG_AND_METRICS_CONFIG_VOLUME_MOUNT = "/opt/cruise-control/custom-config/";
+    protected static final String CONFIG_VOLUME_NAME = "config";
+    /**
+     * Server config file name
+     */
+    public static final String SERVER_CONFIG_FILENAME = "cruisecontrol.properties";
```

Comment:
why are we using the `SERVER_` naming here? The same applies to the annotation below. Isn't this constant and the annotation related to Cruise Control only, so that something like `CRUISE_CONTROL_` is clearer?

Reply:
I wanted to draw the distinction between the configuration of the Cruise Control server application and the Cruise Control broker capacity, since both are Cruise Control configurations. I figured since this fields and annotations are in the Cruise Control class and Cruise Control pod respectively, it would be clear they were apart of Cruise Control!

Reply:
Does that explanation make sense, does it sound reasonable to you?

Reply:
Ok, makes sense. Thanks!